common/defs/bitcoin/ecash.json

```
@@ -0,0 +1,44 @@
+{
+  "coin_name": "eCash",
```

I prefer to keep the name like this.

That way we do not need to change 2 regex-es, only one.

```suggestion
  "coin_name": "Ecash",
```